docs/openeuropa-components.md

```diff
@@ -16,71 +16,69 @@ Below the main, active tracks maintained by the OpenEuropa core team:
 
 ### Editorial track
 
-Maintainers: [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Diogo](https://github.com/dxvargas)
-  
-Components:
 - [OpenEuropa Editorial](https://github.com/openeuropa/oe_editorial)
 
-### Multilingual track
+Maintained by [Hernani](https://github.com/hernani), [Adam](https://github.com/nagyad), [Diogo](https://github.com/dxvargas)\
+Slack Channel [#openeuropa-editorial](http://platform-ec-europa.slack.com/app_redirect?channel=openeuropa-editorial)
```

Comment:
Why put a link to a private Slack in a public document?

Reply:
Good question.
We could add the information that we should have a `@ext.ec.europa.eu` email address to create an account.
![image](https://user-images.githubusercontent.com/186827/62780670-20455000-baae-11e9-8cbb-44c0a6793161.png)

Reply:
why not use something which is open-source instead ;)